### ISSUE TITLE
Issue about the released report

### ISSUE THREAD

**Issue by fyb99** [ToniChopp/ECAMP#7]
Hi, thanks for your amazing work. I find in the pretrain dataset loading period, you write the code as

        csv_path = os.path.join(self.data_root, 'mimic-cxr-2.0.0-entity-llm.csv')
        attn_csv_path = os.path.join(self.data_root, 'mimic-cxr-2.0.0-attn-label.csv')

but the released distilled report file is another file, MIMIC-Vicuna.json. How to get the derived reoprts for pretraining?

**Comment by fyb99** [comment]
BTW, in other issues, i find you said that the MIMIC-Vicuna.json provides the original report, image path and ditilled report. But it seems it does not contain the information which should be provided by 'attn_csv_path', How to deal with it ?

**Comment by ToniChopp** [comment]
> Hi, thanks for your amazing work. I find in the pretrain dataset loading period, you write the code as
> 
> ```
>     csv_path = os.path.join(self.data_root, 'mimic-cxr-2.0.0-entity-llm.csv')
>     attn_csv_path = os.path.join(self.data_root, 'mimic-cxr-2.0.0-attn-label.csv')
> ```
> 
> but the released distilled report file is another file, MIMIC-Vicuna.json. How to get the derived reoprts for pretraining?

Hi, the columns in `mimic-cxr-2.0.0-entity-llm.csv` are `[img_path, report, llm_output]`. You can merge the CSV file containing the image path and original report with our distilled reports in `MIMIC-Vicuna.json` by matching `study_id`.

If you have any further questions about the data, please feel free to let me know!

**Comment by fyb99** [comment]
attn_csv_path = os.path.join(self.data_root, 'mimic-cxr-2.0.0-attn-label.csv')

it seems the provided json file does not contain the information mentioned in the above code, seems like attention masks, which should be provided by 'attn_csv_path', How to deal with it ?

**Comment by ToniChopp** [comment]
> attn_csv_path = os.path.join(self.data_root, 'mimic-cxr-2.0.0-attn-label.csv')
> 
> it seems the provided json file does not contain the information mentioned in the above code, seems like attention masks, which should be provided by 'attn_csv_path', How to deal with it ?

Thanks for your issue. `mimic-cxr-2.0.0-attn-label.csv` contains the [x, y] coordinates of the attention regions derived using GLoRIA. We previously forgot to open-source this file, but it is now available and can be downloaded here: [https://drive.google.com/file/d/1_sv46Mh8vKS7MWgfeAcaZJDvbbMVpmft/view](https://drive.google.com/file/d/1_sv46Mh8vKS7MWgfeAcaZJDvbbMVpmft/view)

**Comment by fyb99** [comment]
thx for your reply
